### ISSUE TITLE
Firefox CSS broken on v119

### ISSUE THREAD

**Issue by hbthen3rd** [hbthen3rd/dotfiles#39]
Seems like tabbar background and selected tab background colors are hard-coded now.
Also, addressbar popup suggestion url.

**Comment by hbthen3rd** [comment]
Seems like an easy fix, but likely won't fix since the current CSS will be replaced with `Pywalfox` once #36 is done anyway.
